trestle/utils/fs.py

```diff
@@ -450,20 +444,14 @@ def allowed_task_name(name: str) -> bool:
     return True
 
 
-def text_files_equal(path_a: pathlib.Path, path_b: pathlib.Path) -> bool:
-    """Determine if files are equal, ignoring newline style."""
-    try:
-        with open(path_a, 'r') as file_a:
-            with open(path_b, 'r') as file_b:
-                lines_a = file_a.readlines()
-                lines_b = file_b.readlines()
-                nlines = len(lines_a)
-                if nlines != len(lines_b):
-                    return False
-                for ii in range(nlines):
-                    if lines_a[ii].rstrip('\r\n') != lines_b[ii].rstrip('\r\n'):
-                        return False
-    except Exception as e:
-        logger.warn(f'Exception comparing file {path_a} to {path_b}.  Return as False. {e}')
-        return False
-    return True
+def model_name_from_href_path(href: pathlib.Path) -> str:
```

Comment:
these model_name_from_href functions either are gone already or will be.  there may be a conflict but no big deal.  the main point is they are no longer needed.  the href command takes care of this.

Reply:
Let me check that may have been a merge issue.